tests/test_suite/run_test_command.py

```diff
@@ -237,7 +240,9 @@ def otx_eval_testing(template, root, otx_dir, args):
     assert os.path.exists(f"{template_work_dir}/trained_{template.model_template_id}/performance.json")
 
 
-def otx_eval_openvino_testing(template, root, otx_dir, args, threshold):
+def otx_eval_openvino_testing(
+    template, root, otx_dir, args, threshold=0.0, criteria={}, reg_threshold=0.10, result_dict={}
```

Comment:
It could be. The reason why I selected this manner is I don't want to make duplicate functions in `run_test_command.py`.
For the `otx_eval_openvino_testing` or such other test functions was implemented as no-return. So, I just followed the way

Reply:
Then how about making function return necessary things instead of modifying `result_dict`? You don't have to make duplicated function also.
And I forget one thing. I think you need to set default value of `criteria` as None. Using dictionary as default value could be dangerous. Please refer [this](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python)

Reply:
> Then how about making function return necessary things instead of modifying `result_dict`? You don't have to make duplicated function also. And I forget one thing. I think you need to set default value of `criteria` as None. Using dictionary as default value could be dangerous. Please refer [this](https://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python)

For the return value, I want to change the code architecture later, not in this PR because we need to get test results within 1 week. 

In the case of dictionary initialization, I'll revise it